src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classNames from 'classnames';
+import { TodosFilter } from '../../types/TodoFilter';
+
+type Props = {
+  filter: TodosFilter,
+  setFilter: (filter: TodosFilter) => void,
+  activeTodosCount: number,
+  hasCompletedTodosCount: number,
+  handleClearCompleted: () => void;
+};
+export const TodoFilter: React.FC<Props> = ({
+  filter,
+  setFilter,
+  activeTodosCount,
+  hasCompletedTodosCount,
+  handleClearCompleted,
+}) => {
+  const itemText = activeTodosCount === 1 ? 'item' : 'items';
```

Comment:
This needs to be worked out:
<img width="564" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/54860641/ee4a6ed5-10f8-41dd-bbda-66cbca8aca7e">

Reply:
When I do npm start everything works correctly
But When I follow the link it doesn't work

Reply:
Now It should work